msteams-platform/tutorials/get-started-yeoman.md

```diff
@@ -45,45 +45,54 @@ Install the Teams Yeoman generator with the following command:
 npm install generator-teams --global
 ```
 
-To install preview versions of the generator, run this command:
+Install the preview version of the generator with the following command:
 
 ```bash
 npm install generator-teams@preview --global
 ```
 
 ## Generate your project
+This section walks you through the steps for generating your project.
 
-Open up a command prompt and create a new directory where you want to create your project and in that directory run the command `yo teams`.
+**To generate your project**
```

Comment:
@KirtiPereira Is this a preferred convention now? I believe the docs validation tool flags this since it's not a semantic heading (H1, H2, etc.). It also repeats the heading above (Generate your project).

Reply:
Hi @heath-hamilton - yes, we are going with this convention, wherever we are calling out the steps. Will check about the validation tool flags, as we not received any warning or faced an issue.